h/api/presenters.py

```diff
@@ -54,7 +54,8 @@ def tags(self):
 
     @property
     def target(self):
-        target = {'source': self.annotation.target_uri}
+        target = {'source': self.annotation.target_uri,
+                  'scope': self.annotation.target_uri_normalized}
```

Comment:
Ahhh, here is where this gets messy. There's no need for this `scope` rubbish to be exposed to clients, and so I'd really rather we didn't start adding stuff to the presenter layer just for the sake of the search index.

Perhaps it would be better to move the responsibility for adding the scope field into `h.api.search.index.index`?